PeerioTechnologies/peerio-desktop
# feat: clicking notification opens the correct DM/room

These three lines to activate the chat I copied from ChatList 😢  Since I don't know what's going on there, suggestions on how to refactor this into one place are welcome.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  

https://peerio.tpondemand.com/entity/7595

#### Manual testing instructions  

0. Put Peerio into background.
1. Wait for notification.
2. Click on it.

What it did before: Peerio activates, but shows in the same place where you left it.

What it should do now: Peerio activates and navigates to the DM or room this notification is coming from.


----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
